include/mme-app/mmeSmDefs.h

```diff
@@ -78,24 +84,33 @@ const uint16_t DDN_FROM_SGW = 110;
 const uint16_t DEL_SESSION_RESP_FROM_SGW = 111; 
 const uint16_t DETACH_ACCEPT_FROM_UE = 112; 
 const uint16_t DETACH_REQ_FROM_UE = 113; 
-const uint16_t ESM_INFO_NOT_REQUIRED = 114; 
-const uint16_t ESM_INFO_REQUIRED = 115; 
-const uint16_t ESM_INFO_RESP_FROM_UE = 116; 
-const uint16_t IDENTITY_RESPONSE_FROM_UE = 117; 
-const uint16_t IMSI_VALIDATION_FAILURE = 118; 
-const uint16_t IMSI_VALIDATION_SUCCESS = 119; 
-const uint16_t INIT_CTXT_RESP_FROM_UE = 120; 
-const uint16_t MB_RESP_FROM_SGW = 121; 
-const uint16_t PURGE_RESP_FROM_HSS = 122; 
-const uint16_t REL_AB_RESP_FROM_SGW = 123; 
-const uint16_t S1_REL_REQ_FROM_UE = 124; 
-const uint16_t SEC_MODE_RESP_FROM_UE = 125; 
-const uint16_t SERVICE_REQUEST_FROM_UE = 126; 
-const uint16_t TAU_REQUEST_FROM_UE = 127; 
-const uint16_t UE_CTXT_REL_COMP_FROM_ENB = 128; 
-const uint16_t ULA_FROM_HSS = 129; 
-const uint16_t VALIDATE_IMSI = 130; 
-const uint16_t END_EVENT = 131;    
+const uint16_t ENB_STATUS_TRANFER_FROM_SRC_ENB = 114; 
```

Comment:
same comment..its difficult to figure out actual change.

Reply:
All of these are created off the state machine JSON, i feel its just easier to look through the list if its in alphabetical order.